e2e/tests/localAuthorityCompliesWithDirectionsAfterSDOIsIssued_test.js

```diff
@@ -0,0 +1,57 @@
+const config = require('../config.js');
+const response = require('../fixtures/response');
+const directions = require('../fixtures/directions');
+
+let caseId;
+
+Feature('Comply with directions');
+
+Before(async (I, caseViewPage, submitApplicationEventPage, sendCaseToGatekeeperEventPage, draftStandardDirectionsEventPage) => {
+  if (!caseId) {
+    await I.logInAndCreateCase(config.swanseaLocalAuthorityEmailUserOne, config.localAuthorityPassword);
```

Comment:
Maybe extract this, as we get further into the case life cycle we would be repeating steps such as this over and over in new tests

Reply:
good idea - more like tech debt cleanup though; we have duplicate code everywhere in other e2e tests which could probably be extracted

Reply:
I'm with Joe, I am not too fussed about extracting this logic here. 29 file changes as is 🙁

Reply:
Yeah it was a sort of thing that isn't required but at some point would be nice